tests/unit/frameworks/pytorch/resnet_serialize_test.py

```diff
@@ -46,23 +58,20 @@ def test_identical(self):
             init_batch_norm=False,
             device=device,
         )
-        # create temp file
-        model_path = tempfile.NamedTemporaryFile(delete=True)
-        # and write the created model
-        torch.save(model.state_dict(), model_path)
 
-        model_copy = create_model(
+        checkpoint_file = save_model(model)
```

Comment:
It would be cleaner to use context manager and pass the file to save_model as a parameter. Something like this:
```
with tempfile.NamedTemporaryFile(delete=True) as checkpoint_file:
    save_model(model, checkpoint_file)
    checkpoint_file.flush()

    model2 = create_model(
         :
         :
```

This way you don't have to worry who opens or closes the file, it is enclosed within the context manager

Reply:
done